tools/SetupFlow/DevHome.SetupFlow/Services/SetupFlowOrchestrator.cs

```diff
@@ -171,12 +170,53 @@ private bool CanGoToPreviousPage()
         return HasPreviousPage && CurrentPageViewModel.CanGoToPreviousPage;
     }
 
+    // Teaching tip needs to be open and visible to show up.

```

Comment:
Can you explain more about why this is? I'm sure there's a good reason to have both these properties but since they're (almost) always in sync it's a little confusing.

Reply:
Unfortunately I don't know the relationship between these two.  I'm going to talk with Windows UX folks to ask why.  But, I did try all combonations including
1. Visibility only
  1. OneTime
  2. OneWay
2. IsOpen only
  1. OneTime
  2. OneWay
3. IsOpen and Visibility
  1. IsOpen:OneTime and Visibility:OneTime
  2. IsOpen:OneTime and Visibility:OneWay
  3. IsOpen:OneWay and Visibility:OneTime
  4. IsOpen:OneWay and Visiblity:OneWay

only the combination above produced the correct results.

Reply:
if the two members should always be in sync, we can just keep one and bind it to all applicable UI properties